x-pack/platform/plugins/shared/streams/server/routes/content/route.ts

```diff
@@ -155,7 +188,40 @@ const importContentRoute = createServerRoute({
   },
 });
 
+const previewContentRoute = createServerRoute({
+  endpoint: 'POST /internal/streams/{name}/content/preview',
+  options: {
+    access: 'internal',
+    summary: 'Preview a content pack',
+    description: 'Returns a json representation of a content pack.',
+    body: {
+      accepts: 'multipart/form-data',
+      maxBytes: MAX_CONTENT_PACK_SIZE_BYTES,
+      output: 'stream',
+    },
+  },
+  params: z.object({
+    path: z.object({
+      name: z.string(),
+    }),
+    body: z.object({
+      content: z.instanceof(Readable),
+    }),
+  }),
+  security: {
+    authz: {
+      enabled: false,
+      reason:
+        'This API delegates security to the currently logged in user and their Elasticsearch permissions.',
```

Comment:
Question: Where in the code called by the handler function are the logged in user/Elasticsearch permissions coming into play? I can't seem to find any scoped clients/etc.

Reply:
Updated in https://github.com/elastic/kibana/pull/217288/commits/7a8723c2f15b7aaaafd44c53edd41c0b053e6baf